handlers/invoiceCron.go

```diff
@@ -0,0 +1,156 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/go-co-op/gocron"
+	"github.com/stakwork/sphinx-tribes/config"
+	"github.com/stakwork/sphinx-tribes/db"
+)
+
+func InitInvoiceCron() {
+	s := gocron.NewScheduler(time.UTC)
+	s.Every(1).Seconds().Do(func() {
+		invoiceCount, _ := db.Store.GetInvoiceCount(config.InvoiceCount)
+
+		if invoiceCount > 0 {
```

Comment:
Yes, it is decremented on line 134 ```// reduce the invoice count
									db.Store.SetInvoiceCount(config.InvoiceCount, invoiceCount-1)```

Reply:
yes but notice we only go into this block when the invoice is settled
https://github.com/stakwork/sphinx-tribes/pull/542/files#diff-348edef07379395dcf013289eef8e7148fef690f5898280c4def5d70dd24bc40R53

what if the invoice is never settled?

Reply:
should we remove the invoice from the list after a certain amount of time, probably when we stop polling would be fine

otherwise the cache can be bloated with unpaid invoices

Reply:
> should we remove the invoice from the list after a certain amount of time, probably when we stop polling would be fine
> 
> otherwise the cache can be bloated with unpaid invoices

ok @kevkevinpal I will add this functionality